lightning/backend/session.go

```diff
@@ -112,8 +112,6 @@ func newSession(options *SessionOptions) *session {
 		txn:  transaction{},
 		vars: vars,
 	}
-	s.vars.GetWriteStmtBufs().BufStore = &kv.BufferStore{MemBuffer: &s.txn}
```

Comment:
This line is used to avoid some allocation and improve performance. Does the updated TiDB not support this?

Reply:
Updated TiDB removed this value, maybe there is other way to support it. I'll check it.

Reply:
Perhaps just check if the flamegraph leading to #274 still happens.